frontend/tests/controllers/UserFilterTest.php

```diff
@@ -51,11 +51,11 @@ public function testMyEvents() {
 
         $login = self::login($identity);
         $response = \OmegaUp\Controllers\User::apiValidateFilter(new \OmegaUp\Request([
-            'filter' => '/user/' . $user->username,
+            'filter' => '/user/' . $identity->username,
```

Comment:
```suggestion
            'filter' => "/user/{$identity->username}",
```

por consistencia con L73

Reply:
Listo